app/src/main/java/org/kiwix/kiwixmobile/utils/AnimationUtils.java

```diff
@@ -61,4 +65,30 @@ public boolean willChangeBounds() {
         (int) (initialHeight / view.getContext().getResources().getDisplayMetrics().density));
     view.startAnimation(animation);
   }
+
+  //rotate animation for closeAllTab FAB
+  public static void rotate(FloatingActionButton v) {
+    float mAngleToRotate = 360f; // rotate 1 round
+    RotateAnimation wheelRotation =
+        new RotateAnimation(0.0f, mAngleToRotate, v.getWidth() / 2.0f, v.getHeight() / 2.0f);
+    wheelRotation.setDuration(50); // rotate 1 round in 50 milliSeconds
```

Comment:
Great work! I think the animation is very fast to be noticeable can you check the material guidelines for the duration of such animations.